apps/challenges/views.py

```diff
@@ -2239,3 +2245,292 @@ def get_challenge_evaluation_cluster_details(request, challenge_pk):
+
+    error_messages = []
+
+    # Check for challenge title.
+    challenge_title = yaml_file_data.get("title")
+    if not challenge_title or len(challenge_title) == 0:
+        message = "Challenge title is missing. Please include."
+        error_messages.append(message)
+
+    # Check for challenge image in yaml file.
+    image = yaml_file_data.get("image")
+    if image and (
+        image.endswith(".jpg")
+        or image.endswith(".jpeg")
+        or image.endswith(".png")
+    ):
+        challenge_image_path = join(
+            base_location, extracted_folder_name, image
+        )
+        if isfile(challenge_image_path):
+            challenge_image_file = read_file_data_as_content_file(challenge_image_path,
+                                                                  "rb", image)
+        else:
+            challenge_image_file = None
+            message = "WARNING: There is no image file."
+            error_messages.append(message)
+    else:
+        challenge_image_file = None
+        message = "ERROR: There is no key for image."
+        error_messages.append(message)
+
+    # check for challenge description file
+    challenge_config_location = join(base_location, extracted_folder_name)
+    is_valid, message = is_challenge_config_yaml_html_field_valid(yaml_file_data,
+                                                                  "description",
+                                                                  challenge_config_location)
+    if not is_valid:
+        error_messages.append(message)
+
+    # check for evaluation details file
+    is_valid, message = is_challenge_config_yaml_html_field_valid(yaml_file_data,
+                                                                  "evaluation_details",
+                                                                  challenge_config_location)
+    if not is_valid:
+        error_messages.append(message)
+
+    # check for terms and conditions file
+    is_valid, message = is_challenge_config_yaml_html_field_valid(yaml_file_data,
+                                                                  "terms_and_conditions",
+                                                                  challenge_config_location)
+    if not is_valid:
+        error_messages.append(message)
+
+    # check for submission guidelines file
+    is_valid, message = is_challenge_config_yaml_html_field_valid(yaml_file_data,
+                                                                  "submission_guidelines",
+                                                                  challenge_config_location)
+    if not is_valid:
+        error_messages.append(message)
+
+    # Check for evaluation script path in yaml file.
+    evaluation_script = yaml_file_data.get("evaluation_script")
+    if evaluation_script:
+        evaluation_script_path = join(challenge_config_location, evaluation_script)
+        # Check for evaluation script file in extracted zip folder.
```

Comment:
` # Check for evaluation script file in extracted zip folder.` -> ` # Check for evaluation script file in extracted zip folder`